webcomponents/tool/src/main/frontend/js/sakai-permissions.js

```diff
@@ -65,6 +65,16 @@ class SakaiPermissions extends SakaiElement {
   get tool() { return this._tool; }
 
   updated(changedProperties) {
+    let inputs = this.renderRoot.querySelectorAll("input[type='checkbox']");
+    inputs.forEach(elem => { 
+      let role = elem.getAttribute('data-role');
+      let perm = elem.getAttribute('data-perm');
+      
+      if(role && perm){
```

Comment:
Please leave a space between between if and the brace. Same for the curly brace on the end.
if (role && perm) {

I am very fussy about these tiny things ... :) I also like a blank line after the function definition if the body has more than one line of code in it. I am fussy, ask Miguel!